site/content/entry/compatability-list-wiki-created.md

```diff
@@ -0,0 +1,15 @@
++++
+date = "2017-06-03T11:30:00-04:00"
+title = "Compatibility List/Wiki Created!"
+author = "Saphiresurf"
+forum = 1603
++++
+
+[Check it Out Here] (https://citra-emu.org/game/)
+
+As game compatibility has broadened and the popularity of Citra has risen there's continued to be a bit of a void for information in terms of what you can actually play with Citra. When you launch a game it's been a bit of a blind test when you launch a game or even when progressing through them unless you do your own research.
```

Comment:
I definitely agree I think this rephrasing is much better! Is there any issue with the original concept of that sentence you had though or do you think it simply needed to be rephrased?